examples/draft-0-10-0/rich/rich.html

```diff
@@ -43,14 +43,13 @@
           this.focus = () => this.refs.editor.focus();
           this.onChange = (editorState) => this.setState({editorState});
 
-          this.handleKeyCommand = (command) => this._handleKeyCommand(command);
+          this.handleKeyCommand = this._handleKeyCommand.bind(this);
```

Comment:
Initially I wrote that with `command` but it was greater than 80 columns. Then I decided to use `bind` because it's literally what we want (have the right `this` inside `_handleKeyCommand` and no need to update in the future if more parameters are added) and because it's also the style in other examples.
That was my rationale but I can update it to your suggestion if you prefer, I don't have a strong opinion here.